ADOPTERS.md

```diff
@@ -8,6 +8,8 @@ has extremely broad production usage as a component of the [Docker engine](https
 stack. Note that this includes any use of the open source [Moby engine project](https://github.com/moby/moby);
 including the Balena project listed below.
 
+**[faasd by OpenFaaS](https://github.com/openfaas/faasd)** - faasd in an Open Source project for serverless functions. It takes the same OpenFaaS components that usually run on Kubernetes and instead launches containers directly on a single host using CNI for networking. It's ideal for edge and for deploying functions without having to think about managing and maintaining Kubernetes.
+
```

Comment:
Not sure about the ordering of the list

Reply:
I checked and "f" comes after Docker and before IBM. 

Just let me know if you need it moving ☝️ 👇